test/test_utils/__init__.py

```diff
@@ -25,7 +25,8 @@
 # Deep Learning Base AMI (Ubuntu 16.04) Version 25.0 used for EC2 tests
 UBUNTU_16_BASE_DLAMI_US_WEST_2 = "ami-0e5a388144f62e4f5"
 UBUNTU_16_BASE_DLAMI_US_EAST_1 = "ami-0da7f2daf5e92c6f2"
-UL_AMI_LIST = [UBUNTU_16_BASE_DLAMI_US_WEST_2, UBUNTU_16_BASE_DLAMI_US_EAST_1]
+PT_GPU_PY3_BENCHMARK_IMAGENET_AMI = "ami-0673bb31cc62485dd"
+UL_AMI_LIST = [UBUNTU_16_BASE_DLAMI_US_WEST_2, UBUNTU_16_BASE_DLAMI_US_EAST_1, PT_GPU_PY3_BENCHMARK_IMAGENET_AMI]
```

Comment:
PT_GPU_PY3_BENCHMARK_IMAGENET_AMI: Is this AMI in us-east-1 or us-east-2?

Reply:
It is in us-east-1. I will add the region info in its name